### PR TITLE
Cve 2019 1002100 minor versions

### PR DESCRIPTION
**Pull request by ferrandinand**
A denial of service vulnerability was reported in kube-apiserver in which authorized users with API write permissions can cause the API server to consume excessive resources while handling a write request.

The issue is medium severity and can be resolved by upgrading the kube-apiserver to v1.12.6, or v1.13.4.

CREATED
[v4.1.1] 
- Update kubernetes to 1.13.4 CVE-2019-1002100

PATCHED
[v3.8.0] WIP
- Update kubernetes to 1.12.6 CVE-2019-1002100

CREATED 
[v3.7.5]
- Update kubernetes to 1.12.6 CVE-2019-1002100

Towards: https://github.com/giantswarm/giantswarm/issues/5506

### CODE REVIEW

File: v_3_7_5/cloudconfig.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
+package v_3_7_4
```
Comment (calvix) [inline v_3_7_5/cloudconfig.go]: this needs to match 3_7_5

Comment (calvix) [reply v_3_7_5/cloudconfig.go]: same for all other files

Comment (mazzy89) [reply v_3_7_5/cloudconfig.go]: right. good catch

File: v_4_0_1/filemanager.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	"os"
+	"path/filepath"
+	"runtime"
+	"text/template"
+
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	version  = "v_4_0_0"
```
Comment (calvix) [inline v_4_0_1/filemanager.go]: this needs to match 4_0_1

File: v_4_0_1/cloudconfig.go [hunk 3]
```diff
@@ -0,0 +1,101 @@
+package v_4_0_0
```
Comment (calvix) [inline v_4_0_1/cloudconfig.go]: this needs to match 4_0_1

Comment (calvix) [reply v_4_0_1/cloudconfig.go]: + all other files

**Comment by corest** [comment]
Overall good. But it is kind of impossible to check what is the actual change. That's why we usually make two PRs:
1. Copy package to new version
2. Make actual change

File: CHANGELOG.md [hunk 4]
```diff
@@ -43,7 +51,7 @@ version directory, and then changes are introduced.
 
 ## [v3.7.3]
 
-### Changed
+### Chgsanged
```
Comment (corest) [inline CHANGELOG.md]: typo?

Comment (calvix) [reply CHANGELOG.md]: :trollface:

Comment (ferrandinand) [reply CHANGELOG.md]: FAIL 
Solved now

**Pull request merged**